tests/nodetree/test_node_unit.py

```diff
@@ -425,30 +466,34 @@ def test_getattr(self, zi):
     def test_getitem(self, path_segments):
         path = join_path(path_segments)
 
-        subnode_mock = Mock()
-        stubs = [
-            MockPartialNode(path_segments[: i + 1]) for i in range(len(path_segments))
-        ]
-        subnode_mock.side_effect = stubs
-
         with patch(
             "labone.nodetree.node.split_path",
             side_effect=lambda x: split_path(x),
+            autospec=True,
         ) as split_patch, patch(
             "labone.nodetree.node.normalize_path_segment",
             side_effect=lambda x: x,
+            autospec=True,
         ) as normalize_patch, patch(
             "labone.nodetree.node.PartialNode.try_generate_subnode",
-            subnode_mock,
+            side_effect=[
+                MockPartialNode(path_segments[: i + 1])
+                for i in range(len(path_segments))
+            ],
+            # TODO adding autospec=True breaks test
```

Comment:
DO not overcomplicate this ... if autospec does not work do not add it but write a comment why autospec does not work